src/common/helpers.ts

```
@@ -145,6 +146,11 @@ export const fetchAndStoreGtagIds = async (): Promise<void> => {
 
 	if (typeof idResult === 'string') {
 		setClientID(idResult, 'google');
+		await LittledataPixel.sendEvent({
```

@igorkol I see fetchAndStoreGtagIds is used in collibrius-s as well. it will throw an error because LittledataPixel is not initialised. Can you wrap this in a try / catch to just swallow the error. i don't want to report from segment setups so it's fine i reckon

---

right! on it